src/components/MetricsPane/charts/AggregateHardCoralCover.jsx

```diff
@@ -57,29 +60,34 @@ export const AggregateHardCoralCover = () => {
     })
     .filter((record) => record !== undefined)
 
-  const markerColors = hardCoralAveragesPerSurvey.map((_, index) => {
-    const colors = chartTheme.aggregateCharts.hardCoralCover.marker
+  const bins = Array.from({ length: (end - start) / binSize }, (_, i) => start + i * binSize)
```

Comment:
In not sure if its better or worse for readability , but you can also do `(,index)`…. you dont _technically_ need a named unused variable. 

Also consider using the full word `index` instead of `i`

Reply:
Am I right that `bins` is storing a list of bin _start_ values? If so, can ti be called `binStartValues` or something like that?

Reply:
1) mmm, you can't do `(,index)` , it is syntax error
2) I change the bins name to `binStartValues`